safe-global/safe-wallet-web
# fix: Add proposers feature flag [SW-488]

## What it solves

Resolves SW-488

## How this PR fixes it

- Adds a new feature flag `PROPOSERS` to turn off add/edit/delete

## How to test it

1. Open the settings page with at least one proposer added
2. Observe add/edit/delete are visible

## Screenshots
<img width="824" alt="Screenshot 2024-11-06 at 10 25 49" src="https://github.com/user-attachments/assets/fe633115-9930-4543-ba3d-eea687d65ded">

## Checklist
